testdata/ce/go.tpl

```diff
@@ -0,0 +1,67 @@
+{{- $suffix := randAlphaNum 5 | lower -}}
+apiVersion: codeengine.cloud.ibm.com/v1beta1
+kind: JobRun
+metadata:
+  name: {{ .Name }}-{{ $suffix }}
```

Comment:
Could we please get rid of `{{ .Name }}` and replace it with a fixed string, like `experiment-run`.